lib/chef-dk/component_test.rb

```diff
@@ -18,6 +18,15 @@
 require 'chef-dk/exceptions'
 require 'chef-dk/helpers'
 
+# https://github.com/bundler/bundler/issues/4368
+module Bundler
+  class SpecSet
+
+    alias_method :size, :length
+
+  end
+end
```

Comment:
Alternatively, we could pin to rubygems 2.6.1 for now. Or we could keep the monkey patch but add some conditions to it, so that e.g., it only applies if `Bundler::SpecSet` is defined and does not have a `size` method.